chains/docker-compose.yml

```diff
@@ -1,11 +1,28 @@
 version: "3.3"
 services:
   development:
-    image: trufflesuite/ganache-cli:v6.12.2
+    image: ghcr.io/foundry-rs/foundry:stable
+    entrypoint: ["anvil"]
+    command:
+      [
+        "--host","0.0.0.0",
+        "--port","8545",
+        "--chain-id","5777",
```

Comment:
It seems that `5777` was used by Ganache, but we don't need to keep it. The default value (31337) should be fine.